examples/couchdb_load/test_1.py

```diff
@@ -1,6 +1,3 @@
-import pytest
```

Comment:
It is necessary to synchronize real-world examples with the examples in the documentation.